comps/llms/vllm-openvino/build_vllm_openvino.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Copyright (c) 2024 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+git clone --branch openvino-model-executor https://github.com/ilya-lavrenov/vllm.git
```

Comment:
Suggest we check with OV team if they plan to merge their changes to vLLM mainline and the details thereof to identify OV as the model executor. We may have changes accordingly.

Reply:
Agreed. Better to get a more official code source (if merged), not a private branch.

Reply:
We are in discussion with the Openvino team on this one.